bayesianflows.tex

```diff
@@ -75,63 +73,72 @@ \section*{Main text}
-% WDP: This bit comes across as weaker than intended (to my eyes, which are a bit biased because I get a bit antsy about axiomatically assuming anything in stats) because it almost (but doesn't quite, I recognise) imply that only frequentists assume something, and I think it might get people's backs up a bit. Perhaps, instead, you could lean away from the Bayes vs. frequency bit here and point out that parts of this approach are agnostic with regards to frequency vs. Bayes (that makes it sounds like we're saying something very new) but that to get the full advantages of it you have to use Bayes. Lean into the "learn to calibrate and so infer better" angle more strongly? 
 
-The workflow we recommend helps avoid these problems. It provides an organized sequence of steps to explicitly build and challenge a model through simulation. We believe this is easier and more straightforward to do with a Bayesian approach and thus outline the workflow assuming a Bayesian approach, but we suggest a similar approach could, and should, be extended to other statistical inference methods. % Most ecologists are trained primarily in frequentist methods, which are focused on calibration (this is is why training in frequentist methods includes so much discussion of significance, power, mean squared error, and the like). Bayesian methods typically focus on inference, but can be equally well calibrated (see Step 1, below). 
 
-To better understand our workflow, we provide a very brief overview of some of the fundamentals of Bayesian methods that is inherently incomplete and, by design, not very technical. This section can be skipped for those who feel already well-versed, and can be augmented for those who are new to Bayesian approaches \citep[for example,][]{statrethink,BDA,regotherstories}.
 
-\emph{A brief review of statistical inference using Bayesian approaches}
+\section{A brief overview of Bayesian methods and their advantages}
+To better understand our workflow, we provide a very brief overview of some of the fundamentals of Bayesian methods that is inherently incomplete and, by design, not very technical. This section can be skipped for those who feel already well-versed, and can be augmented for those who are new to Bayesian approaches \citep[for example,][]{statrethink,BDA,regotherstories}.
 
-Probability is often defined as ``the long-term frequency with which something happens.'' We would expect, for example that if we tossed a coin 100 times we would see roughly 50 heads.  In this case we would say that the probability of tossing a coin and getting a `head' is $\frac{50}{100}$, equivalent to 50\% or $\frac{1}{2}$. At the same time we wouldn’t be very surprised if we observed 49 or even 55 heads, although we would be surprised if we saw 99.
+\subsection{Bayesian and frequentist probability and inference}
```

Comment:
I basically moved most of this section to the supp to save words. Also, since Mike was fairly happy with our language here I avoided changing it so much (when I moved it to the supp).